keps/sig-api-machinery/1164-remove-selflink/README.md

```diff
@@ -176,6 +177,22 @@ require more explanation.
 
 [deprecation-policy]: https://kubernetes.io/docs/reference/using-api/deprecation-policy/
 
+#### GA
+
+- Feature remained disabled by default for 4 releases
+- Feature disabled by default in cloud-providers: GKE, OpenShift
```

Comment:
So I checked for versions >= 1.19 and we're not touching the `SelfLink` featuregate on EKS either. Should just be the default in k/k.